index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Next Web</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/7b838c6d68.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <div class="header-nav-1">
+                <div class="links-left">
+                    <a href="#"><span class="white-text">News</span></a>
+                    <a href="#">Events</a>
+                    <a href="#">Business</a>
+                    <a href="#">AMAs</a>
+                    <a href="#">Spaces</a>
+                    <a href="#">Terms & Condition</a>
+                </div>
+                <div class="links-right">
+                    <a href="#">About</a>
+                    <a href="#">Advertise</a>
+                    <a href="#">Jobs</a>
+                    <a href="#">Contact</a>
+                </div>
+            </div>
+            <div class="header-nav-2">
+                <div class="logo-wrap"><img class="logo-image" src="img/tnw-logo.png" alt="logo-image"></div>
+                <span class="invisible-links">
+                    <a href="#">LATEST</a>
+                    <a href="#">HARD FORK</a>
+                    <a href="#">PLUGGED</a>
+                    <a href="#">README</a>
+                    <a href="#">GROWTH QUARTERS</a>
+                </span>
+                <div class="">
+                    <a href="#"><i class="fas fa-search"></i></a>
+                    <a href="#"><i class="fas fa-bars"></i></a>
+                </div>
+            </div>
+            <div class="header-nav-3">
+                <a href="">
+                    <p>
+                        Calling young digital talent in the Netherlands. be recognzed in the T500 <i
+                            class="fas fa-arrow-right"></i>
+                    </p>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="content-container flex">
+            <div class="content-info first-image flex flex-1">
+                <a href="#"><img src="img/logo-transparent.png" alt="big logo"></a>
+                <h1>Opal cuts off your apps’ internet access to help you stop doomscrolling</h1>
+                <span class="date">Tuesday - January 26, 2021</span>
```

Comment:
- [ ] [Optional] This text is not fully visible. perhapse you could add some styling to it. Example `padding: 5px 0 0 
![image](https://user-images.githubusercontent.com/64360974/114018450-e8d80980-9875-11eb-891f-cb8f510ed9ac.png)